web/src/lottie-player.model.ts

```diff
@@ -0,0 +1,98 @@
+import { LitElement } from 'lit';
+import { property } from 'lit/decorators.js';
+
+// Define library version
+interface LibraryVersion {
+  THORVG_VERSION: string
+}
+
+// Define file type which can be exported
+enum ExportableType {
+  GIF = 'gif',
+}
+
+export abstract class LottiePlayerModel extends LitElement {
+  @property({ type: Boolean })
+  public direction: number = 1;
+
+  @property({ type: Number })
+  public speed: number = 1.0;
+
+  @property({ type: Number })
+  public totalFrame?: number;
+
+  @property({ type: Number })
+  public currentFrame?: number;
+
+  @property({ type: Boolean })
+  public loop: boolean = false;
+
+  /**
+   * Start playing animation.
+   */
+  public abstract play (): void
+
+  /**
+   * Pause animation.
+   */
+  public abstract pause (): void
+
+  /**
+   * Stop animation.
+   */
+  public abstract stop (): void
+
+  /**
+   * Start playing animation.
+   */
+  public abstract load (src: string | object): void
+
+  /**
+   * Seek to a given frame
+   *
+   * @param frame Frame number to move
+   */
+  public abstract seek (frame: number): void
+
+  /**
+   * Sets the repeating of the animation.
+   *
+   * @param value Whether to enable repeating. Boolean true enables repeating.
+   */
+  public abstract setLooping (value: boolean): void
+
+  /**
+   * Animation play direction.
+   *
+   * @param value Direction values.
+   */
+  public abstract setDirection (value: number): void
+
+  /**
+   * Sets animation play speed.
+   *
+   * @param value Playback speed.
```

Comment:
In the doc, It needs the default value info and expected range (ie. 0.1 ~ )

Reply:
til now, I think any positive number can be player back speed. so I commented like this, but feel free to let me know if there is any specific speed range of thorvg animation.

```js
* @param value Playback speed. (any positive number)
```